src/gui/main.cpp

```diff
@@ -385,25 +405,35 @@ QSettings::setDefaultFormat(CkbSettings::Format);
     if(background)
         shm_str = nullptr;
 
-    if(isRunning(shm_str) && !QtCreator){
+    if(!background && isRunning(shm_str) && !QtCreator){
```

Comment:
Won't this allow for multiple instances fighting each other under the same user (possibly messing up the config too)?